typespec/Applications.Core/extensions.tsp

```diff
@@ -24,46 +24,46 @@ model Extension {}
 
 @doc("Kubernetes namespace extension of a environment/application resource.")
 model KubernetesNamespaceExtension extends Extension {
-  @doc("The kind of the resource.")
+  @doc("The kind of extension being used. Must be ‘kubernetesNamespace’")
   kind: "kubernetesNamespace";
 
-  @doc("The namespace of the application environment.")
+  @doc("The namespace where all application-scoped resources generate Kubernetes objects.")
   `namespace`: string;
 }
 
 @doc("Kubernetes metadata extension of a environment/application resource.")
 model KubernetesMetadataExtension extends Extension {
-  @doc("The kind of the resource.")
+  @doc("The kind of extension being used. Must be 'kubernetesMetadata'")
   kind: "kubernetesMetadata";
 
-  @doc("Annotations to be applied to the Kubernetes resources output by the resource")
+  @doc("The Kubernetes annotations to set on your application and its resources")
```

Comment:
line 39 versus line 42.. use of 'your application' versus 'the application'. I think we mostly do not use possessive pronouns. Pls review. I would request standardizing on one approach.